src/drivers/imx/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ add_local_sources(sof
 	interrupt.c
 	ipc.c
 	timer.c
+	sdma.c
 )
 
```

Comment:
TODO: Add selectable config for SDMA. e.g: CONFIG_IMX_SDMA. This can be done later.

Reply:
I think I can do this in the same PR that will have the implementation.